lib/miq_automation_engine/engine/miq_ae_engine.rb

```diff
@@ -49,6 +49,44 @@ def self.deliver_queue(args, options = {})
     end
   end
 
+  def self.deliver_synchronous(args)
+    log_header = "MIQ(#{name}.deliver_synchronous)"
+    options       = {:action => "Automate Request", :userid => 'system'}
```

Comment:
@gmcculloug 
Thats already being done via the user_id attribute
https://github.com/ManageIQ/manageiq/blob/master/app/models/resource_action.rb#L22

This userid is not necessary since the user_id is being set. I have removed it